### PR TITLE
[TESTONLY]

### PR DESCRIPTION
**Pull request by chaodaiG**

### CODE REVIEW

File: scripts/presubmit.sh [hunk 1]
```diff
@@ -86,6 +86,9 @@ if [ -n "${OUT}" ]; then
   exit 1
 fi
 
+while true; do docker ps >>"${ARTIFACTS}/docker.out"; done &
```
Comment (sethvargo) [inline scripts/presubmit.sh:89]: Let's sleep for like 2-3s here just so we don't super-spew.

Comment (sethvargo) [inline scripts/presubmit.sh:89]: Can we make this per-run (like is there a build ID or something we can use)?

Comment (chaodaiG) [reply scripts/presubmit.sh:89]: This actually saves to a sub-directory of where the job log is located, it's pretty well separated from run to run

**Comment by sethvargo** [comment]
/close 

we fixed it

**Pull request closed without merging**